.github/workflows/ci.yml

```diff
@@ -7,30 +7,35 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby_version: [2.5.x, 2.6.x, 2.7.x]
-
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
+        exclude:
+          - gemfile: rails_master.gemfile
+            ruby_version: 2.4.x
+          - gemfile: rails_6.0.0.gemfile
+            ruby_version: 2.4.x
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby
       uses: actions/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby_version }}
-    - name: Download CodeClimate reporter
+    - name: Update rubygems when testing with Ruby 2.4.x
+      if: startsWith(matrix.ruby_version, '2.4')
+      run: |
+        gem update --system --no-document
+    - name: Before build
       run: |
+        sudo apt-get install libsqlite3-dev
         curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
         chmod +x ./cc-test-reporter
         ./cc-test-reporter before-build
       env:
-        CC_TEST_REPORTER_ID: get_a_test_reporter_id_from_code_climate
+        CC_TEST_REPORTER_ID: 33a23b18b12b65ab41da2e9b664dd576d2cc03a04da268473108570fc72cfa62
     - name: Build and test
       run: |
         gem install bundler:1.17.3
-        bundle update
-        bundle install --jobs 4 --retry 3
-        bundle exec rake code_analysis
-        bundle exec rspec
-    - name: Report to CodeClimate
-      run: |
-        ./cc-test-reporter after-build --exit-code 0
-      env:
-        CC_TEST_REPORTER_ID: get_a_test_reporter_id_from_code_climate
```

Comment:
You should report back to code climate

Reply:
it's not working. check YAAF, it doesn't have this part either. I created a ticket to fix it